source/GlassView.Export/Export.cs

```diff
@@ -0,0 +1,97 @@
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Mathematics;
+using BenchmarkDotNet.Portability.Cpu;
+using BenchmarkDotNet.Reports;
+using GlassView.Core.Models;
+
+namespace GlassView.Export;
+
+public static class Export
+{
+    public static BenchmarkSummary Map(Summary summary)
+    {
+        var titleSegments = summary.Title.Split('-');
+        var timestamp = ParseTimestamp(titleSegments[1..]);
```

Comment:
This seems really a complicated way of getting the timestamp. It's a pitty dotnet chose to deserialize its data in such a way...